blueprints/dumb/files/__test__/components/__name__.spec.js

```diff
@@ -0,0 +1,8 @@
+import React from 'react'
+import <%= pascalEntityName %> from 'components/<%= pascalEntityName %>/<%= pascalEntityName %>'
```

Comment:
@SpencerCDixon it would be nice to _selectively_ wrap files in folders, since I don't really want ducks/containers/etc. to be wrapped in an extra folder, only components and views.